cmd/network_chain_revert_launch.go

```diff
@@ -12,10 +12,19 @@ import (
 // to revert a launched chain.
 func NewNetworkChainRevertLaunch() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "revert-launch [launch-id]",
-		Short: "Revert launch of a network",
-		Args:  cobra.ExactArgs(1),
-		RunE:  networkChainRevertLaunchHandler,
+		Use: "revert-launch [launch-id]",
+		Short: `The revert launch command reverts the previously scheduled launch of a chain.
```

Comment:
Shouldn't it be in `Long` field instead ?

Reply:
https://github.com/ignite/cli-plugin-network/pull/10/commits/119bb4af3c3f4f95d32336e5b0cf58e12931c735